src/components/visualization-bar/index.css

```diff
@@ -0,0 +1,291 @@
+.hx-visualization-bar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: var(--visual-content-margin);
+  height: auto;
+}
+
+.hx-visualization-bar-header {
+  order: 1;
+}
+
+.hx-visualization-bar-count {
+  order: 2;
+}
+
+.hx-visualization-bar-body {
+  order: 3;
+}
+
+.hx-visualization-bar-content {
+  order: 4;
+}
+
+.hx-visualization-bar .hx-hidden {
+  display: none;
+}
+
+.hx-visualization-bar-header,
+.hx-visualization-bar-body {
+  width: 100%;
+  display: flex;
+}
+
+.hx-visualization-bar-body {
+  flex-direction: column;
+}
+
+.hx-visualization-bar .hx-header-small {
+  margin-bottom: var(--visual-content-margin);
+}
+
+.hx-visualization-bar.disabled .hx-visualization-bar-title .hx-header-small {
+  color: var(--visual-disabled-text-color);
+}
+
+.hx-visualization-bar.disabled .hx-visualization-bar-count {
+  display: none;
+}
+
+.hx-visualization-bar.disabled .hx-visualization-bar-content {
+  color: var(--visual-disabled-text-color);
+}
+
+.hx-visualization-bar-title,
+.hx-visualization-bar-count {
+  display: flex;
+}
+
+.hx-visualization-bar-title {
+  width: 70%;
+}
+
+.hx-visualization-bar-count {
+  width: 30%;
+  flex-direction: row-reverse;
+}
+
+.hx-visualization-bar-fill-wrapper {
+}
+
+.hx-visualization-bar-fill {
+  background-color: var(--visual-healthy-na-color);
```

Comment:
I can still see quite a few variables used in `index.css` here, these styles should be declared inside `theme.css` (i.e. don't use variables inside `index.css`)